sympy/ntheory/factor_.py

```diff
@@ -2225,3 +2225,62 @@ def is_amicable(m, n):
         return False
     a, b = map(lambda i: divisor_sigma(i), (m, n))
     return a == b == (m + n)
+
+
+def eva(number, operation):
+    """Generates digits of a number."""
```

Comment:
This docstring is inaccurate. The function either computes the sum or the product of the decimal digits of a number.

Also this function should probably be two separate functions: one for multiply and one for add.